### ISSUE TITLE
Add creation and update dates to meta tags in detail pages

### ISSUE THREAD

**Issue by jvillalobos** [mozilla/addons-frontend#5916]
The `<head>` of detail pages should include the following tags:

`<meta name="date" content="2018-07-24T16:01:00+0200" />`
`<meta name="last-modified" content="2018-07-24T16:01:00+0200" />`

The `date` value should correspond to the add-on's creation time, and the `last-modified` value should correspond to the last updated time.

/cc @rraue

**Comment by Bharghvi** [comment]
Hi, I can pick up this issue. But need to clarify some doubts as I am new to this project.

**Comment by willdurand** [comment]
@Bharghvi sorry, I am working on it! I let you find another issue in [this list]().

This issue needs an API change, filed at: https://github.com/mozilla/addons-server/issues/9564

**Comment by willdurand** [comment]
> @willdurand You are right, Google and Bind do not use date and last-modified meta tags. But both use last-modified in sitemaps, so we need this data anyways when we want to improve our sitemaps. Writing them into the head makes sense for analytics, so we can attribute pages and their creation and update dates. And this data helps at the end to improve Seo. But for technical Seo its just important that all pages have last-modified in the sitemap.
> 
> But for the sitemap it is very important to have it, so crawlers do not use at our big website too much crawling-budget for unnecessary pages.

Thanks for your reply!

If I follow correctly, we still want to add those two `meta` tags for analytics, right? (analytics as in Google Analytics or other tools?)

**Comment by willdurand** [comment]
API has landed btw.

**Comment by rraue** [comment]
@willdurand Yes, that's the easiest way to read this data so we won't need to pull and match two data sources. As well we will need this data-point once we add schema js+ld to the page.

**Comment by willdurand** [comment]
>  As well we will need this data-point once we add schema js+ld to the page.

Thanks. Talking about jsonld, I'd need your input on https://github.com/mozilla/addons-frontend/issues/4129 😸

**Comment by ioanarusiczki** [comment]
@willdurand 
I verified this issue on AMO dev with FF62 (Win10).
What I've noticed is that no matter if the first version of the add-on has been disabled or deleted, the meta data will display the date of that first version submitted. 
I don't see it as a problem but let me know if it's alright.

I also submitted a new version to check the last update too and metadata `<meta data-react-helmet="true" name="last-modified" content="2018-10-17T15:08:32Z">` displays it correctly.

An example 
![metadata for creation and update](https://user-images.githubusercontent.com/33448286/47097875-ee22e300-d23a-11e8-8801-595d6379b11b.png)
